tensorrt_llm/_torch/custom_ops/flashinfer_custom_ops.py

```
@@ -56,23 +97,25 @@ def _(input: torch.Tensor, weight: torch.Tensor,
-                          eps,
-                          enable_pdl=get_env_enable_pdl())
+        _call_with_nfs_retry(fused_add_rmsnorm,
+                             input,
+                             residual,
+                             weight,
+                             eps,
+                             enable_pdl=get_env_enable_pdl())
 
     @torch.library.custom_op("trtllm::flashinfer_gemma_fused_add_rmsnorm",
                              mutates_args=("input", "residual"))
     def flashinfer_gemma_fused_add_rmsnorm(input: torch.Tensor,
```

Ruff (0.15.10)

[error] Function argument `input` is shadowing a Python builtin

(A002)